addons/website/static/src/scss/website.scss

```diff
@@ -274,7 +274,9 @@ $-header-nav-link-height: $nav-link-height;
     @include o-apply-colors('footer');
     @include o-apply-colors('footer-custom');
     @include o-add-gradient('footer-gradient');
-
+    &.o_footer_slideout {
+        overflow-y: hidden;
+    }
```

Comment:
If I don't put this and a footer element has an animation where it comes from the bottom of the page, there is a flicker of the page height

Reply:
> QSM, I hesitated here to reproduce the _toggleOverflowXHidden mechanism for the footer slideout Y axis. I decided not to do it because I don't see any inconvenience in hiding the Y overflow all the time on these footers.

I would prefer this ;) There are always problems with overflow hidden at all times :wink: In two month after this, I would get a bug report such as "I have a dropdown in my footer and it is cut in half when I open it" ;)